sources/packages/backend/libs/integrations/src/sfas-integration/sfas-files/sfas-application-record.ts

```diff
@@ -117,4 +117,158 @@ export class SFASApplicationRecord extends SFASRecordIdentification {
   get applicationCancelDate(): Date | null {
     return parseDate(this.line.substring(137, 137 + 8));
   }
+  /**
+   * The unique application number assigned to this application (application_alias.application_no).
+   */
+  get applicationNumber(): number | null {
+    return parseInteger(this.line.substring(145, 155));
```

Comment:
It's not a blocker, no need to change it, but I have seen the two formats of executing the substring along the files (not this one is specific), `145, 155` vs `127, 127 + 10`. I personally prefer the first one but I do not have a strong opinion other than "we should do it in the same way" 😉